Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/ICalibrationParser.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    /// <summary>
+    /// Interface to allow multiple parsers to attempt to parse calibration data.
+    /// </summary>
+    public interface ICalibrationParser
```

Comment:
>ICalibrationParser [](start = 21, length = 18)

does this end up as a monobehaviour? it seems like maybe we should add an example using the CalibrationData class that is now in this repo.

Reply:
Will definitely get to this but not with this change. Need to get an example calibration from Chris to test ingesting it with CalibrationData